pythonforandroid/bootstraps/sdl2/build/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -1063,7 +1111,43 @@ public void surfaceChanged(SurfaceHolder holder,
         mWidth = width;
         mHeight = height;
         SDLActivity.onNativeResize(width, height, sdlFormat, mDisplay.getRefreshRate());
-        Log.v("SDL", "Window size:" + width + "x"+height);
+        Log.v("SDL", "Window size: " + width + "x" + height);
```

Comment:
You used Log.v(TAG, ...) in most places now, but not here. Is that an oversight maybe?

Reply:
What I did in this PR is just replacing the SDLActivity with the one that comes with SDL 2.0.4 and then merging some changes by the python-for-android team back in. 

Here, the SDL guys dediced to not use the `TAG` constant and to just keep the raw `"SDL"` string. Don't ask me why :P

Reply:
Haha okay, that explains it. Well it's no biggie, it's not like the output prefix is vital. Just something that stumped me